lib/CreepPlayers/Behaviors/MoveToExitHallwayDumb.js

```diff
@@ -0,0 +1,44 @@
+
+var MoveUtils = require('../../Utils/MoveUtils');
+var Base = require('./Base');
+var Direction = require('../../GameObjects/Direction');
+
+/**
+ *
+ * @param creepController @type CreepController
+ * @constructor
+ */
+function MoveToExitDumb(creepController) {
+    this.cc = creepController;
+}
+
+MoveToExitDumb.prototype = Object.create(Base.prototype);
+
+MoveToExitDumb.prototype.act = function() {
+    var toExit = this.cc.senseDirectionToExit();
+    var toPlayer = this.cc.getDirectionToPlayer();
+
+    var numWalls = 0;
+    if (!this.cc.canMove(Direction.NORTH)) {
```

Comment:
Maybe check for North/South and East/West

If you have North/East you won't be blocking anything!